enterprise-cfengine-guide/federated-reporting.markdown

```diff
@@ -23,13 +23,33 @@ configure and connect the Superhub and Feeder hubs. For Feeder hubs with an
 earlier version than 3.14.0 some manual steps must be taken. Links to these
 are provided at each stage of installation and setup that follows.
 
+* [Requirements][Federated Reporting#Requirements]
 * [Installation][Federated Reporting#Installation]
 * [Setup][Federated Reporting#Setup]
 * [Operation][Federated Reporting#Operation]
 * [API Setup][Federated Reporting#API Setup]
 * [Disable Feeder][Federated Reporting#Disable Feeder]
 * [Uninstall][Federated Reporting#Uninstall]
 
+## Requirements ##
+
+If your hub will have SELinux enabled you must install the `semanage` command.
+This allows Federated Reporting policy to manage the trust between the superhub and
+feeder hubs.
```

Comment:
I suppose technically it allows policy to manage the cftransport user but I was trying to talk at a higher level. @roschern thoughts?

Reply:
Well, as long as we clearly state what it does, I think that is more important than listing all the ways it achieves it. 
As long as it doesn't become a problem for users, if log files or other info contradicts or is confusing in light of this documentation.